src/drivers/px4fmu/fmu.cpp

```diff
@@ -2128,7 +2168,51 @@ PX4FMU::peripheral_reset(int ms)
 		ms = 10;
 	}
 
-	board_peripheral_reset(ms);
```

Comment:
@kd0aij 

All the ifdefs have been removed on master. The board source now provides functions or macros that have a layer of abstraction.

The src/board/<name>_init.c provided ```board_peripheral_reset(int ms)``` and src/board/<name>_spi.c files provide ```board_spi_reset(int ms)``` so the code below belongs in https://github.com/PX4/Firmware/blob/master/src/drivers/boards/px4fmu-v4/px4fmu_init.c#L189

Note: On master it is OK to use the px4_arch_ prefixes in src/boards/  nuttx_v3 will uses the stm32_prefix
I will port the changes to nuttx_v3 when this make it to master.